logging/README.md

```diff
@@ -257,6 +257,48 @@ The script creates the namespace into which the components are deployed. By defa
 Updates in place are supported. To update, re-run the
 `deploy_logging_open.sh` script to install the latest versions of all components, indexes, and dashboards.
 
+## Important Information about Kibana in the New Release
+
+### Overview
+
+**Notes:**
```

Comment:
I think it is either NOTES or OVERVIEW, but not both.  I like the bullet list presentation and would prefer to keep it rather than present that information in paragraph form.

Reply:
This section and its formatting follow SAS Style Guidelines.